docs/source/using/addons.adoc

```diff
@@ -255,6 +255,8 @@ By default, this is not allowed in OpenShift.
 | Creates a user named 'admin' and assigns the *cluster-admin* role to it.  
 | registry-route
 | Creates an edge terminated route for the OpenShift registry.
+| max-pod
```

Comment:
Hi @robin-owen can you please take a look, i am not very satisfied with the description :-P

Reply:
What's there is true, but doesn't help the reader understand what the add-on does. Something along the lines of "adjusts the maximum number of pods allocated for each CPU core" could work. :)

Reply:
Aha.. now there's a shortcoming in this addon, to set the number of pods/CPU core, 'pods-per-core' value has to be modified. both the properties have an effect on the number of pods a node can run.

ref: https://docs.openshift.com/container-platform/3.5/admin_guide/manage_nodes.html#admin-guide-max-pods-per-node